firmware/quadruna/BMS/src/cubemx/Inc/FreeRTOSConfig.h

```diff
@@ -51,13 +51,12 @@
 #if defined(__ICCARM__) || defined(__CC_ARM) || defined(__GNUC__)
 #include <stdint.h>
 extern uint32_t SystemCoreClock;
-void            xPortSysTickHandler(void);
 #endif
 #ifndef CMSIS_device_header
 #define CMSIS_device_header "stm32h7xx.h"
 #endif /* CMSIS_device_header */
 
-#define configENABLE_FPU 0
+#define configENABLE_FPU 1
```

Comment:
I don't actually know much about this, but I was wondering why there was a change with the FPU.

Reply:
I notiched it wasn't enabled in FreeRTOS config, so enabled it since we need it for SoC calculations

Reply:
Okay, I see. Makes sense makes sense